pkg_ci.es.Rmd

```diff
@@ -33,79 +33,96 @@ Los paquetes de R deben tener CI para todos los sistemas operativos (Linux, Mac
+Mira, por ejemplo, [este archivo de configuración](https://github.com/r-lib/usethis/blob/2c52c06373849d52f78a26c5a0e080f518a2f825/inst/templates/appveyor.yml#L13).
+AppVeyor usará la caché automáticamente si lo configuras con `usethis::use_appveyor()`.
 
-- Activa [construcciones rodantes](https://www.appveyor.com/docs/build-configuration/#rolling-builds).
+- Activa [rolling builds](https://www.appveyor.com/docs/build-configuration/#rolling-builds).
 
-Ya no transferimos los proyectos de AppVeyor a la cuenta de AppVeyor de ropensci, así que después de transferir tu repo a la organización de GitHub "ropensci" de rOpenSci, la insignia será `[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/ropensci/pkgname?branch=master&svg=true)](https://ci.appveyor.com/project/individualaccount/pkgname)`.
+Ya no transferimos los proyectos de AppVeyor a la cuenta de AppVeyor de rOpenSci, así que después de transferir tu repo a la organización de GitHub "ropensci", la etiqueta será `[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/ropensci/pkgname?branch=master&svg=true)](https://ci.appveyor.com/project/individualaccount/pkgname)`.
 
 ### Circle CI (Linux y Mac OSX)
 
 [Circle CI](https://circleci.com/) es utilizado, por ejemplo, por el paquete rOpenSci [`bomrang`](https://github.com/ropensci/bomrang) como servicio de integración continua.
 
-## Cobertura de pruebas {#cobertura}
+## Cobertura de tests {#coverage}
 
-La integración continua también debe incluir la notificación de la cobertura de las pruebas a través de un servicio de pruebas como [Codecov](https://codecov.io/) o [Coveralls](https://coveralls.io/).
+La integración continua también debe incluir información de la cobertura de los tests a través de un servicio de testing como [Codecov](https://codecov.io/) o [Coveralls](https://coveralls.io/).
 
 Recomendamos utilizar Codecov.
-Para activar Codecov para tu repo, ejecuta `usethis::use_github_action("test-coverage")` para crear un archivo `.github/workflows/test-coverage.yaml`.
-También tienes que dar acceso a Codecov a tu repositorio de github, ver [Guía de inicio rápido de Codecov](https://docs.codecov.com/docs/quick-start) para saber cómo configurar el acceso.
-A continuación, añade una insignia de estado de Codecov en la parte superior de tu README.md, consulta [Insignias de estado de Codecov](https://docs.codecov.com/docs/status-badges).
+Para activar Codecov para tu repo, ejecuta `usethis::use_github_action("test-coverage")` el cual va a crear un archivo `.github/workflows/test-coverage.yaml`.
+También tienes que dar acceso a Codecov a tu repositorio de GitHub, ver [la guía de inicio rápido de Codecov](https://docs.codecov.com/docs/quick-start) para saber cómo hacerlo.
+Luego añade una etiqueta de estado de Codecov en la parte superior de tu README.md, puedes consultar [Status Badges](https://docs.codecov.com/docs/status-badges) en la documentación de Codecov.
```

Comment:
I think we keep "Status Badges" because it makes reference to an external document in English. Similar to what we are doing for blog post titles. 

But I'm also ok if we decide to translate it.

Reply:
I will check what I have about this on the Guide and add this as a decision (It is clear that my first intention it is to translate, but we can keep the original decision)